runtime/binding-kafka/src/test/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/ClientFetchIT.java

```diff
@@ -514,7 +514,7 @@ public void shouldReceiveMessagesWithHeadersSkipManyFilter() throws Exception
     @Specification({
         "${app}/filter.sync/client",
         "${net}/filter.sync/server"})
-    public void shouldFetchMergedMessageLatest() throws Exception
+    public void shouldFetchLatestMessages() throws Exception
```

Comment:
Usually we name the test methods according to the spec scenarios.
In this case, i think that would be `shouldFetchFilterSync`, works?